.github/workflows/cd-pipeline.yml

```diff
@@ -65,35 +65,8 @@ jobs:
       - name: Pause for manual approval
         run: echo "Deployment paused for manual approval."
 
-  merge-to-release:
-    needs: approval-deploy-staging
-    # These permissions are needed to write to the release branch
-    permissions:
-      contents: write
-    runs-on: ubuntu-latest
-    steps:
-      - name: Checkout release branch
-        uses: actions/checkout@v4
-        with:
-          ref: release
-          fetch-depth: 0
-          # Fine-grained PAT with contents:write and workflows:write
-          # scopes
-          token: ${{ secrets.CD_PAT }}
```

Comment:
@k-macmillan I think we can remove this secret now? I'm not sure if anything else uses it, but grep isn't showing it being used anywhere in the repo after this was removed.

```
coreycarvalho@Mac:~/code/notification-api [1969-deploy-from-main|✔] $ grep -R CD_PAT .github/
coreycarvalho@Mac:~/code/notification-api [1969-deploy-from-main|✔] $ grep -R CD_PAT 
coreycarvalho@Mac:~/code/notification-api [1969-deploy-from-main|✔] $ 
```